sympy/physics/vector/tests/test_frame.py

```diff
@@ -720,3 +721,13 @@ def test_unit_dyadic():
     F = ReferenceFrame('F', indices=['1', '2', '3'])
     assert N.u == N.xx + N.yy + N.zz
     assert F.u == F.xx + F.yy + F.zz
+
+
+def test_pickle_frame():
+    N = ReferenceFrame('N')
+    A = ReferenceFrame('A')
+    A.orient_axis(N, N.x, 1)
+    A_C_N = A.dcm(N)
+    N1 = pickle.loads(pickle.dumps(N))
+    A1 = tuple(N1._dcm_dict.keys())[0]
+    assert A1.dcm(N1) == A_C_N
```

Comment:
The assertion only checks that these two matrices are the same but you pickle the reference frame. Shouldn't you be checking reference frames before and after?

Reply:
ok, i don't think this hurts anything, so let's merge

Reply:
At the minimum you could open an issue or add to one of the existing issues an explanation of the further complexity.